src/Symfony/Bundle/SecurityBundle/Resources/config/security_authenticator_access_token.php

```diff
@@ -69,10 +69,11 @@
             ->args([
                 abstract_arg('signature algorithm'),
                 abstract_arg('signature key'),
+                abstract_arg('audience'),
+                abstract_arg('issuers'),
                 service('logger')->nullOnInvalid(),
                 service('clock'),
                 'sub',
```

Comment:
what about also moving the claim before the logger? I think that's make more sense to me